fenix/app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt

```diff
@@ -116,54 +104,7 @@ class BookmarksTest {
         }
     }
 
-    @Test
-    fun verifyBookmarkButtonTest() {
-        val defaultWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 1)
-
-        navigationToolbar {
-        }.enterURLAndEnterToBrowser(defaultWebPage.url) {
-        }.openThreeDotMenu {
-        }.bookmarkPage {
-        }.openThreeDotMenu {
-            verifyEditBookmarkButton()
-        }
-    }
-
-    @Test
-    fun addBookmarkTest() {
```

Comment:
verifyBookmarkButtonTest and addBookmarkTest could be merged into one and add a TR test for them.